drydock/plugin.py

```diff
@@ -1,11 +1,56 @@
 from glob import glob
 import os
+import yaml
 import pkg_resources
 
+from tutor import env as tutor_env
+from tutor import config as tutor_config
 from tutor import hooks
 
+import logging
+
 from .__about__ import __version__
 
+INSTALLATION_PATH = f'{os.getcwd()}/env'
+
+INIT_JOBS_SYNC_WAVE = -100
+
+
+def get_init_tasks():
+    """Return the list of init tasks to run."""
+    init_tasks = list(hooks.Filters.CLI_DO_INIT_TASKS.iterate())
+    config = tutor_config.load(os.getcwd())
```

Comment:
Is [this](https://github.com/overhangio/tutor/blob/v15.3.7/tutor/commands/jobs.py#L326) a more standard way to get the config? by getting the current context first? It used used for Palm and Quince as well